lib/govuk_ab_testing/rspec_helpers.rb

```diff
@@ -0,0 +1,39 @@
+module GovukAbTesting
```

Comment:
I can see how we could remove some duplication by introducing this change. However, I would rather do itwhen we introduce another test helper class, so we have a 3rd example on how to implement `with_variant`. I don't know enough about its usage at this stage to know if that would be a good way forward or not.